pyproject.toml

```diff
@@ -26,7 +26,7 @@ pytz = "^2023.3.post1"
 tqdm = "^4.66.1"
 black = { version = "^23.10.1", optional = true }
 pytest = { version = "^7.4.3", optional = true }
-llama-index = "0.9.13"
+llama-index = "^0.9.13"
```

Comment:
@sarahwooders for some reason w/o adding the carrot `^` llama-index kept getting downgraded by poetry when I `poetry add`'d `fastapi`